src/v3/src/OktaSignIn/index.ts

```diff
@@ -14,6 +14,7 @@ import { OktaAuth, OktaAuthOptions, Tokens } from '@okta/okta-auth-js';
 import pick from 'lodash/pick';
 import { h, render } from 'preact';
 import { TinyEmitter as EventEmitter } from 'tiny-emitter';
+import { setCookieUserAuthenticated } from 'src/util';
```

Comment:
Sometimes we see issues when absolute paths are used, I would recommend using a relative path for this import.

Reply:
fixed: https://github.com/okta/okta-signin-widget/commit/5abc64b4b5a05e743554b7552c095b3fd5078210